### PR TITLE
ASM-9315 Network scale-up only shows a single used network - missing all the other available networks

### PR DESCRIPTION
**Pull request by sushilrai**
Included VLAN information for standard port-group which was missing in the existing inventory for vCenter.

### CODE REVIEW

**Comment by aimers1975** [comment]
@gavin-scott  or @kaushalgala  could one of you review?  Also, don't close 9315, as @AlexDron  has work todo after.

File: bin/discovery.rb [hunk 1]
```diff
@@ -226,6 +228,14 @@ def collect_vds_portgroup_attributes(portgroup)
   {:vlan_id => vlan_id}
 end
 
+def collect_portgroup_attributes(network_obj)
+  network = network_obj.host[0].configManager.networkSystem.networkInfo.portgroup.select { |x| x.spec.name == network_obj.name }
+  vlan_id = network.first.spec.vlanId
```
Comment (kaushalgala) [inline bin/discovery.rb]: minor nit: add a space here

**Comment by kaushalgala** [comment]
@aimers1975 I have a tiny comment which is not functional, but am ok to merge as-is

**Comment by sushilrai** [comment]
@ripienaar please review.

**Pull request merged**